Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -148,14 +148,14 @@ class OpenSearchEngine: NSObject, NSCoding {
  * OpenSearch spec: http://www.opensearch.org/Specifications/OpenSearch/1.1
  */
 class OpenSearchParser {
-    private let pluginMode: Bool
+    fileprivate let pluginMode: Bool
 
     init(pluginMode: Bool) {
         self.pluginMode = pluginMode
     }
 
-    func parse(file: String, engineID: String) -> OpenSearchEngine? {
-        let data = NSData(contentsOfFile: file)
+    func parse(_ file: String, engineID: String) -> OpenSearchEngine? {
+        let data = try? Data(contentsOf: URL(fileURLWithPath: file))
 
         if data == nil {
```

Comment:
This should be the catch of a `do{}catch{}` block on `try Data(contentsOf: URL(fileURLWithPath: file))`